scripts/cat/pelts.py

```diff
@@ -253,7 +253,20 @@ def __init__(
         self.screen_scale = scripts.game_structure.screen_settings.screen_scale
 
         # converting old pose numbers into names
-        if isinstance(adol_sprite, int):
+        if any(
+            [
+                True
+                for x in [
+                    newborn_sprite,
+                    kitten_sprite,
+                    adol_sprite,
+                    adult_sprite,
+                    senior_sprite,
+                    para_adult_sprite,
+                ]
+                if isinstance(x, int)
+            ]
+        ):
```

Comment:
```suggestion
        if any(
            isinstance(x, int) or x is None
            for x in [
                newborn_sprite,
                kitten_sprite,
                adol_sprite,
                adult_sprite,
                senior_sprite,
                para_adult_sprite,
            ]
        ):
```
* I think this order is a little more clear
* Probably also want to run conversions if any sprites are missing
* Apparently, according to pylint, the list comprehension inside of `any` is not required ([use-a-generator / R1729](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/use-a-generator.html))